tfjs-core/src/ops/fused/conv2d.ts

```diff
@@ -139,8 +140,11 @@ function fusedConv2d_<T extends Tensor3D|Tensor4D>({
                result, activation, preluActivationWeights, leakyreluAlpha) as T;
   }
 
-  const $x = convertToTensor(x, 'x', 'conv2d', 'float32');
-  const $filter = convertToTensor(filter, 'filter', 'conv2d', 'float32');
+  const $x = convertToTensor(
+      x, 'x', 'conv2d', env().getBool('FLOAT16') ? 'float16' : 'float32');
```

Comment:
Please use `engine().backend.supportFloat16()` to check. And add `env().getBool('FLOAT16')` in `supportFloat16()`.